.travis.yml

```diff
@@ -7,6 +7,8 @@ cache:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
 deploy:
+  on:
+    branch: master
```

Comment:
**thought (non-blocking):* I think it's _usually_ a good idea to keep pipeline changes in a separate PR, so they can be isolated if necessary. I do not always follow this advice to the letter myself, I do occasionally regret that though 😄